### PR TITLE
Enable robot_manager.py to communicate with go1 controller to synchronize task reset

### PR DESCRIPTION
**Pull request by aqubits**

### CODE REVIEW

**Comment by linhdoan8** [comment]
@voshch pls test and merge

**Comment by voshch** [comment]
resolve conflicts first

File: task_generator/task_generator/tasks/base_task.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
+import os
```
Comment (voshch) [inline task_generator/task_generator/tasks/base_task.py:1]: undo

**Review by voshch — CHANGES_REQUESTED** [review]

**Comment by voshch** [comment]
This adds hard-coded logic in the task manager; push this to a separate branch (e.g. arena-rosnav:go1) and keep that branch synchronized with master.

**Pull request closed without merging**